LEGO1/realtime/matrix4d.inl.h

```diff
@@ -312,6 +314,7 @@ int Matrix4::Invert(Matrix4& p_mat)
 		}
 
 		if (copy[i][i] < 0.001f && copy[i][i] > -0.001f) {
+			// FAILURE from mxtypes.h
```

Comment:
Not sure if we want to import `mxtypes.h` here, or move `SUCCESS` and `FAILURE` elsewhere, or leave it like this. I think importing `mxtypes.h` may cause a dependency cycle somewhere.